### ISSUE TITLE
Migrade notes on code signing from applicatiln-services repo?

### ISSUE THREAD

**Issue by rfk** [mozilla-mobile/shared-docs#80]
Over in the application-services repo we have some notes on how to get your apks signed via autograph:

  https://github.com/mozilla/application-services/blob/91a10d8/docs/howtos/signing-android-applications.md

This is very specific to building an android app, and feels out of place with the other documentation in that repo. Would you be interested in merging some or all of that content over to an appropriate doc in this android-specific repo instead?

I'm happy to do the legwork if you can give me some pointers on whether and how it would best fit in with the material that's already here.

**Comment by mcomella** [comment]
> This is very specific to building an android app, and feels out of place with the other documentation in that repo. Would you be interested in merging some or all of that content over to an appropriate doc in this android-specific repo instead?

Yes! Though I'm not sure how many teams use autograph (at least FFES does), I think it'd be great to share these notes.

> I'm happy to do the legwork if you can give me some pointers on whether and how it would best fit in with the material that's already here.

@rfk imo, I think the best thing to do would be create a new file in the Android directory (e.g. `android/signing-with-autograph.md`) and link to it from the [CONTRIBUTING_code doc](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md), perhaps in a new "Releasing" section at the bottom? It's not set in stone though so if you have better ideas, let me know!

Thanks for contributing! 😄 

---

By the way, on FFES, [we have a script](https://github.com/mozilla-mobile/firefox-echo-show/blob/master/tools/sign_release.sh) to run these commands, which abstracts away the need to know the  autograph commands. 🤔 Future thinking: I wonder if we should add a "sign with autograph" task to our shared gradle plugin so teams could just run that instead of running the commands directly.
